platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
I can't remember if we talked about this, but most of the validation warnings and errors for the reuters set seem to be null. I think you mentioned using a different data sample?